src/main/scala/org/broadinstitute/dsde/rawls/acl/AccessLevel.scala

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.dsde.rawls.acl
+
+import org.broadinstitute.dsde.rawls.RawlsException
+
+object AccessLevel extends Enumeration {
+  val UnknownUser = Value(0)
+  val None = Value(1)
+  val Read = Value(2)
+  val Write = Value(3)
+  val Owner = Value(4)
+
+  def toGoogleString(v: Value): String = {
+    v match {
+      case Owner => "OWNER"
+      case Write => "WRITER"
+      case Read => "READER"
+      case None => "NONE"
+      case UnknownUser => "UNKNOWN USER"
+      case _ => throw new RawlsException(s"invalid AccessLevel [${v}]")
+    }
+  }
+
+  def fromGoogleString(s: String): Value = {
+    s match {
+      case "OWNER" => Owner
+      case "WRITER" => Write
+      case "READER" => Read
```

Comment:
GCS only specifies these three values only.  If there is no access, there's no ACL entry.  "None" and "Unknown" may turn out to be the same.

https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls